frontend/website/src/components/StatusBadge.re

```diff
@@ -0,0 +1,102 @@
+module Styles = {
+  open Css;
+  let wrapper = (bgColor, fgColor) =>
+    style([
+      backgroundColor(bgColor),
+      color(fgColor),
+      width(`rem(10.)),
+      borderRadius(`px(5)),
+      display(`inlineFlex),
+      justifyContent(`center),
+    ]);
+  let link =
+    style([
+      textDecoration(`none),
+      display(`inline),
+      margin2(~v=`zero, ~h=`rem(0.5)),
+    ]);
+};
+
+let url = "https://status.codaprotocol.com";
+let apiPath = "/api/v2/summary.json";
+
+type component = {
+  id: string,
+  name: string,
+  status: string,
+};
+type response = {components: array(component)};
+
+external parseStatusResponse: Js.Json.t => response = "%identity";
+
+type service = [ | `Summary | `Network | `Faucet | `EchoBot | `GraphQLProxy];
+
+type status =
```

Comment:
these are the other states fyi
<img width="331" alt="Screen Shot 2020-02-12 at 7 49 56 PM" src="https://user-images.githubusercontent.com/50593560/74399817-ea708380-4dd0-11ea-85a8-c211c5e5bc1b.png">